journal/templates/bundles/LiberoPatternsBundle/patterns/content-header.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '@!LiberoPatterns/patterns/content-header.html.twig' %}
```

Comment:
This is an example of replace a single template, then _extending_ it. So the whole thing isn't replaced. (Though could be if you want.)